lib/modules/hosts/raddit.js

```diff
@@ -7,12 +7,13 @@ modules['showImages'].siteModules['raddit'] = {
 			value: true,
 			type: 'boolean'
 		},
-		'autoplay sidebar playlister': {
-			dependsOn: 'show playlister toggle',
-			description: 'Start playing the radd.it sidebar embed when opening it',
-			type: 'boolean',
-			value: true
-		},
+		// No longer supported.
```

Comment:
May as well just delete these lines unless you intend to add them back soonish.  We've got git history.